pysph/tools/ipy_viewer.py

```diff
@@ -540,3 +541,294 @@ def _save_figure_handler(self, change):
                 )
                 break
         self._widgets.save_figure.value = ""
+
+
+class ParticleArrayWidgets3D(object):
+
+    def __init__(self, particlearray, particles_type):
+        self.particles_type = particles_type
+        self.scalar = widgets.Dropdown(
+            options=[
+                'None'
+            ]+particlearray.output_property_arrays,
```

Comment:
PEP8, space around +

Reply:
Yep, I wonder why pep8 did not catch that.